packages/patternfly-4/react-core/src/components/Progress/ProgressContainer.js

```diff
@@ -0,0 +1,71 @@
+import React, { Fragment } from 'react';
+import styles from '@patternfly/patternfly-next/components/Progress/styles.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import { CheckCircleIcon, TimesCircleIcon } from '@patternfly/react-icons';
+import ProgressBar from './ProgressBar';
+
+export const ProgressMeasureLocation = {
+  outside: 'outside',
+  inside: 'inside',
+  top: 'top',
+  none: 'none'
+};
+
+export const ProgressVariant = {
+  danger: 'danger',
+  success: 'success',
+  info: 'info'
+};
+
+const propTypes = {
+  /** Progress component DOM ID. */
+  id: PropTypes.string.isRequired,
+  /** Progress title. */
+  title: PropTypes.string,
+  /** Type of progress status. */
+  variant: PropTypes.oneOf(Object.keys(ProgressVariant)),
```

Comment:
I think you want Object.values instead and also add these to the enumValues object in your docs.js file so the enums are correctly listed in the docs

Reply:
Example: https://github.com/patternfly/patternfly-react/pull/626/files#diff-95ac1dca454c81c21f004d3812e13dc2R18